app/views/categories/_content_items.html.erb

```diff
@@ -1,7 +1,15 @@
 <ol class="category-detail__list">
-  <% contents.each do |item| %>
+  <% contents.take(6).each do |item| %>
```

Comment:
Why take 6? Can we move this to a method on controller and move this magic number as constant?